sdc/rabbit/consumers.py

```diff
@@ -66,26 +67,22 @@ def connect(self):
 
         """
 
-        count = 1
         no_of_servers = len(self._rabbit_urls)
 
         while True:
-            server_choice = (count % no_of_servers) - 1
+            server_choice = (self._count % no_of_servers) - 1
 
             self._url = self._rabbit_urls[server_choice]
```

Comment:
I dont anticipate server choice exceeding len(self._rabbit_urls), however if it ever does then I dont think we would detect it with the current logging self._rabbit_urls[server_choice] could raise an exception. Is it possible to log number of servers, server_choice and attempt before we call self._rabbit_urls[server_choice] ? that way if we ever do get that situation it will be clear in the logs